README-source.md

```diff
@@ -64,10 +64,10 @@ For NVM on Mac (via [homebrew](http://brew.sh/)):
 
 ```bash
 brew install nvm
-nvm install v4.3.2
+nvm install v6.10.2
 ```
 
-You can then either swap to that version with `nvm use v4.3.2`, or do `nvm exec LAMBDA_VERSION zapier test` so you can run tests without having to switch versions while developing.
```

Comment:
You shouldn't remove `LAMBDA_VERSION` here, IIRC that's how we do the version replace